torch/_torch_docs.py

```diff
@@ -169,27 +168,25 @@ def merge_dicts(*dicts):
     >>> torch.add(a, 20)
     tensor([ 20.0202,  21.0985,  21.3506,  19.3944])
 
-.. function:: add(input, alpha=1, other, out=None)
+.. function:: add(input, other, alpha=1, out=None)
 
-Each element of the tensor :attr:`other` is multiplied by the scalar
-:attr:`alpha` and added to each element of the tensor :attr:`input`.
-The resulting tensor is returned.
-
-The shapes of :attr:`input` and :attr:`other` must be
-:ref:`broadcastable <broadcasting-semantics>`.
+Each element of the tensor ``other`` is multiplied by the scalar
+``alpha`` and added to each element of the tensor ``input``.
 
 .. math::
     \text{{out}} = \text{{input}} + \text{{alpha}} \times \text{{other}}
 
-If :attr:`other` is of type FloatTensor or DoubleTensor, :attr:`alpha` must be
-a real number, otherwise it should be an integer.
+If the result of the addition between ``input`` and ``other`` is:
+- floating point, ``alpha`` must be floating point or integral
```

Comment:
fyi on the 1.3 branch we said "float or int" because this is (expected to be) a python value, and python doesn't differentiate float/double. I just realized it is possible to pass a single-value tensor here. "integral" could imply boolean, which isn't allowed in this case. I guess the differentiated "integral or boolean" below makes it clearer that this line refers to non-boolean integral types.